### ISSUE TITLE
Comments are not as wide as the runs table

### ISSUE THREAD

**Issue by ihodes** [hammerlab/cycledash#669]
![](https://d1zjcuqflbd5k.cloudfront.net/files/acc_204365/1gBv8?response-content-disposition=inline;%20filename=Screen%20Shot%202015-06-01%20at%2017.34.00.png&Expires=1433194525&Signature=H5kAf6oY3N-XtGkLBlh3KgpNGADMPVEiAhSQQbojuKuJM4wtA4EVHBbvRI0jeRmPwI2TED4U5Pf~k-~BgchUBpzVQgfd4KIsF9st~JE~dVfuy3ewjx2dMnTmv6cZtbj5lVF1NUz6rZJ9SwGVXajtCzTBL4aiykUw8p18GYxStHY_&Key-Pair-Id=APKAJTEIOJM3LSMN33SA)

**Comment by jaclynperrone** [comment]
Yeahh. The 100% width makes them way too wide. To the point where it's hard to follow. I'm going to be working on a redesign for it anyway. This first deploy was just more about getting it onto a Bootstrap grid

**Comment by ihodes** [comment]
Sound good! I will close in favor of https://github.com/hammerlab/cycledash/issues/651
